README.md

```diff
@@ -173,6 +179,98 @@ Whether or not to force push a reference with different commit history before th
 code-suggester pr -o foo -r bar -d 'description' -t 'title' -m 'message' --git-dir=.
 ```
 
+## Action
+
+### create a pull request
+Opens a GitHub Pull Request against the upstream primary branch with the provided git directory. By default the git directory is the same as the `$GITHUB_WORKSPACE` directory.
```

Comment:
This is looking good to me. Did you set up a testing repo to play with the action.

I'm trying to picture what my workflow would look like as a contributor to a repository. Would I for the repository and it would already have this action within it, and then this action would make changes on behalf of another user.

I think seeing it in action would really help drill home the value to me.

Reply:
I have a repo setup at TomKristie/HelloWorld but it uses my access token. I'll set something up more official. I was also thinking about embedding a gif in the readme which I'll try to do before I leave.